github_actions_report.html.erb

```diff
@@ -1,80 +1,103 @@
-<html><title>Github Action Report</title>
+<html>
+<title>Github Action Report</title>
 <head>
-  <script src='./web_libraries/jquery.min.js' type='text/javascript'></script>
-  <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css'>
-  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
-  <script>
-  $(document).ready(function(){
-    $('[data-toggle="tooltip"]').tooltip();
-  });
+  <link rel="stylesheet" href="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css">
+  <script src='https://code.jquery.com/jquery-3.5.1.js'></script>
+  <script src='./web_libraries/sorttable.js'></script>
+  <link rel='stylesheet' href='./web_libraries/bootstrap.min.css'>
+  <script src='./web_libraries/DataTables/datatables.js'></script>
+  <link rel='stylesheet' href='./web_libraries/DataTables/datatables.css'>
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+  <script src="github_action_report.js"></script>   
+  <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+  <script type="text/javascript">
+  google.charts.load("current", {packages:["corechart"]});
+    function drawChart() {
+      var data = google.visualization.arrayToDataTable([
+        ['Job Status', 'Total'],
+        ['Success', <%= result_hash.map { |x| x[:total_successes] }.reduce(0) { |sum, num| sum + num } %>],
+        ['Failure', <%= result_hash.map { |x| x[:total_failures] }.reduce(0) { |sum, num| sum + num } %>]
+      ]);
+      var options = {
+        title: 'Runs Overview',
+        slices: [{ color: 'green'}, { color: 'red'}],
+        pieHole: 0.4,
+      };
+      var chart = new google.visualization.PieChart(document.getElementById('donutchart'));
+      chart.draw(data, options);
+    }
+     google.charts.setOnLoadCallback(drawChart);
   </script>
-<!-- Google Analytics -->
-<script>
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-ga('create', 'UA-157301458-1', 'auto');
-ga('send', 'pageview', { 'page': location.pathname + location.search + location.hash});
-ga('set', 'anonymizeIp', true);
-</script>
-<!-- End Google Analytics -->
 </head>
 <body>
 <h1 style="text-align:center;">Github Actions Report</h1>
+<center><div id="donutchart" style="width: 900px; height: 500px;"></div></center>
```

Comment:
using a fixed dimension might cause a couple of issues, can you use relative dimensions here ?